### PR TITLE
Add allowed OAuth Tenants

### PR DESCRIPTION
**Pull request by vincent-stytch**

### CODE REVIEW

File: stytch/b2b/api/organizations.py [hunk 1]
```diff
@@ -122,6 +124,8 @@ def create(
           - allowed_mfa_methods: An array of allowed MFA authentication methods. This list is enforced when `mfa_methods` is set to `RESTRICTED`.
           The list's accepted values are: `sms_otp` and `totp`.
 
+          - oauth_tenant_jit_provisioning: (no documentation yet)
+          - allowed_oauth_tenants: (no documentation yet)
```
Comment (logan-stytch) [inline stytch/b2b/api/organizations.py:128]: Can we add docs?

File: stytch/consumer/models/crypto_wallets.py [hunk 2]
```diff
@@ -22,17 +22,17 @@ class SIWEParams(pydantic.BaseModel):
...
       - not_before: The time when the signed authentication message will become valid. Defaults to the current time. All timestamps in our API conform to the RFC 3339 standard and are expressed in UTC, e.g. `2021-12-29T12:33:09Z`.
-      - message_request_id: A system-specific identifier that may be used to uniquely refer to the sign-in request.
+      - message_request_id: A system-specific identifier that may be used to uniquely refer to the sign-in request. The `message_request_id` must be a valid pchar according to RFC 3986 definitions.
     """  # noqa
 
     domain: str
     uri: str
     resources: List[str]
-    chain_id: Optional[int] = None
+    chain_id: Optional[str] = None
```
Comment (logan-stytch) [inline stytch/consumer/models/crypto_wallets.py:35]: Oooooh, what happened here? This is technically a breaking change.

Comment (logan-stytch) [reply stytch/consumer/models/crypto_wallets.py:35]: I see it's related to SIWE... has this been released yet?

**Review by logan-stytch — CHANGES_REQUESTED** [review]

File: stytch/consumer/models/crypto_wallets.py (hunk 2 above)
Comment (ollie-stytch) [reply stytch/consumer/models/crypto_wallets.py:35]: Jenn was talking about this [here](https://stytchio.slack.com/archives/C04LCF8GJQ6/p1723145031432859)

Comment (logan-stytch) [reply stytch/consumer/models/crypto_wallets.py:35]: If it's unreleased, this is fine as a MINOR update

Comment (jennifer-stytch) [reply stytch/consumer/models/crypto_wallets.py:35]: Yup it's unreleased

Comment (vincent-stytch) [reply stytch/consumer/models/crypto_wallets.py:35]: Let's release the SIWE stuff first and I'll rebase off of that

**Review by logan-stytch — APPROVED** [review]
Last thing: bump the MINOR version in `version.py`

**Comment by logan-stytch** [comment]
> Last thing: bump the MINOR version in `version.py`
Oh nice, you did it as I was commenting haha

**Pull request merged**